docs/self-managed/operate-deployment/operate-configuration.md

```diff
@@ -73,23 +74,27 @@ The following configuration is required to enable multi-tenancy in Operate:
 | camunda.operate.multiTenancy.enabled | Activates the multi-tenancy feature within Operate. | false         |
 
 :::caution
-To ensure seamless integration and functionality, the multi-tenancy feature should also be enabled across all associated components. Find more information, including links to individual component configuration on the [multi-tenancy concepts page](/self-managed/concepts/multi-tenancy.md).
+To ensure seamless integration and functionality, the multi-tenancy feature should also be enabled across all associated components so users can view any data from tenants for which they have authorizations configured in Identity. Find more information, including links to individual component configuration on the [multi-tenancy concepts page](/self-managed/concepts/multi-tenancy.md).
 :::
 
-If multi-tenancy is enabled across components, users are allowed to view any data from tenants for which they have authorizations configured in Identity.
-
-If multi-tenancy is disabled in Operate, all users are allowed to view data from the `<default>` tenant only and no data from other tenants.
-
-If multi-tenancy is enabled in Operate but disabled in Identity (or Identity is unreachable for other reasons), users will not have any tenant authorizations in Operate
-and will not be able to access the data of any tenants in Operate.
-
 The same rules apply to the [Operate API](../../apis-tools/operate-api/overview.md#multi-tenancy).
 
 ### Securing Operate - Zeebe interaction
 
 While executing user operations, Operate communicates with Zeebe using the Zeebe Java client. For Zeebe to know whether operations are allowed to be executed
 in terms of tenant assignment, Operate - Zeebe connection must be secured. Check the list of environment variables to be provided in the [Zeebe documentation](../../zeebe-deployment/security/client-authorization/#environment-variables).
 
+### Troubleshooting multi-tenancy in Operate
+
+If users can view data from the `<default>` tenant only and no data from other tenants, multi-tenancy is not enabled in Operate. Refer to the configuration instructions above.
+
+<!-- If a user gets to this section via a Google search, maybe link to MT concepts page, helm chart, so they know they don't need to enabled MT for each component unless they can't use Helm -->
```

Comment:
This is likely something we want to consider for all component docs in MT - routing users back to the concepts page and that really nice, single flag for enabling MT with Helm